crates/swc_ecma_minifier/tests/benches-full/three.js

```diff
@@ -1683,15 +1683,15 @@ function(global, factory) {
         }
         var _proto = Layers.prototype;
         return _proto.set = function(channel) {
-            this.mask = 1 << channel | 0;
```

Comment:
AFAIK `| 0` is a performance optimization (also used by asm.js) so I'm not sure if it's fine to remove it

Reply:
I concluded it's fine. Can you rebase?